.github/workflows/test.yml

```diff
@@ -39,15 +39,15 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python37
+    - name: Set up Python39
       uses: actions/setup-python@v2
       with:
-        python-version: 3.7
+        python-version: 3.9
 
     - name: Install Dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8==2.4.0
+        pip install flake8==5.0.4
```

Comment:
I'm curious what this change is for; is there a specific lint error we are now getting from Py-3.7 / flake8-2?